### DISCUSSION TITLE
export all not exporting all data in powergrid 4.8 when using collection data source

### DISCUSSION THREAD

**Discussion by ashek1412** [Power-Components/livewire-powergrid#1487] [Q&A] [answered]
export all not exporting all data in power grid 4.8 when using collection data source. it only exports data shown in first page.
Can anyone help

**Comment by marineusde** [comment]
Same here, I opened an issue

https://github.com/Power-Components/livewire-powergrid/issues/1490

I fixed it in a pull request:

https://github.com/Power-Components/livewire-powergrid/pull/1491

**Reply by ashek1412** [reply]
@zwhhz , can u export all collection data to excel ?

**Reply by marineusde** [reply, marked as answer]
Yes, I fixed it in the branch. Wait for the next update, if the pull request is merged.

**Reply by ashek1412** [reply]
its not working in version 4.8

**Reply by marineusde** [reply]
I fixed it in version >= v5.4.9, cause I cant test it in 4.x, please upgrade :smiley:

**Reply by ashek1412** [reply]
unfortunately I can't change the version, is there anything else i can do ?

**Reply by marineusde** [reply]
Yes, you can test and merge my changes into the 4.x branch:

https://github.com/Power-Components/livewire-powergrid/pull/1491/commits/de17e4f66ecb1eab9b1c70a03e99a0a4b92d9a5b
